### ISSUE TITLE
Mars local cluster running code error

### ISSUE THREAD

**Issue by sznariOsmosis** [mars-project/mars#1949]
source code:

```
import mars.tensor as mt
from mars.deploy.local import new_cluster
from mars.session import new_session

def main():
    cluster = new_cluster()

    # new cluster will start a session and set it as default one
    # execute will then run in the local cluster
    a = mt.random.rand(10, 10)
    a.dot(a.T).execute()

    # cluster.session is the session created
    (a + 1).execute(session=cluster.session)

    # users can also create a session explicitly
    # cluster.endpoint needs to be passed to new_session
    session2 = new_session(cluster.endpoint)
    (a * 2).execute(session=session2)

if __name__ == "__main__":
    main()
```

erro msg:
Traceback (most recent call last):
  File "src/gevent/greenlet.py", line 906, in gevent._gevent_cgreenlet.Greenlet.run
  File "mars/actors/pool/gevent_pool.pyx", line 1407, in mars.actors.pool.gevent_pool.Dispatcher.on_receive
  File "mars/actors/pool/gevent_pool.pyx", line 1411, in mars.actors.pool.gevent_pool.Dispatcher.on_receive
  File "mars/actors/pool/gevent_pool.pyx", line 1370, in mars.actors.pool.gevent_pool.Dispatcher._on_receive_result
  File "mars/actors/pool/gevent_pool.pyx", line 1387, in mars.actors.pool.gevent_pool.Dispatcher._on_receive_result
AttributeError: 'NoneType' object has no attribute 'put'
2021-01-29T03:23:50Z <Greenlet at 0x7ff355505848: <built-in method on_receive of mars.actors.pool.gevent_pool.Dispatcher object at 0x7ff3555ed408>(b'\x03\x01 \x00\x00\x00\x00\x00\x00\x00k\x01\xc7\n)> failed with AttributeError

**Comment by qinxuye** [comment]
Duplicate with #1941 .

I have pointed out that actually execution succeeded, the error is irrelevant.

```python
import mars.tensor as mt
from mars.deploy.local import new_cluster
from mars.session import new_session

def main():
    cluster = new_cluster()

    # new cluster will start a session and set it as default one
    # execute will then run in the local cluster
    a = mt.random.rand(10, 10)
    print(a.dot(a.T).execute())

    # cluster.session is the session created
    print((a + 1).execute(session=cluster.session))

    # users can also create a session explicitly
    # cluster.endpoint needs to be passed to new_session
    session2 = new_session(cluster.endpoint)
    print((a * 2).execute(session=session2))

    print('succeed, error after this is irrelevant'

if __name__ == "__main__":
    main()
```

Just run the code.

**Comment by qinxuye** [comment]
Anyway, we will fix the irrelevant error messages, after the coming fix, you would never see them.

**Comment by qinxuye** [comment]
Could be closed when #1785 is done.
